lab-bundle-spec.md

```diff
@@ -288,6 +288,11 @@ reference               | displayed as
 [DOMAIN].admin_password | copyable text
 
 
+
+
+
+#### BumperBoats Resources
```

Comment:
BumperBoats is an internal name and not something that I think we should publicize (similar to how Navy does not show up anywhere on this page).

I would either remove this header or rename it to something along the lines of `Virtual Machine Resources`.

Reply:
Makes sense, though it sort of renews my vague ongoing confusion about the intended audience/purpose of this document.